samigo/samigo-app/src/webapp/jsf/qti/chooseExportType.jsp

```diff
@@ -88,10 +88,10 @@ function getSelectedType(qtiUrl, cpUrl, emtUrl, e2mt){
       <p class="sak-banner-warn"><h:outputText value="#{authorImportExport.cp_message}"/></p>
     </div>
     <h:panelGroup layout="block">
-     <h:outputText value="#{authorImportExport.choose_export_type}"/>
-     <t:selectOneRadio id="exportType" layout="spread" value="1">
-       <f:selectItem itemLabel="#{authorImportExport.qti12}" itemValue="1"/>
+     <h:outputLabel value="#{authorImportExport.choose_export_type}"/>
```

Comment:
To properly connect the label to the radio, you need to add the `for=""` attribute here:

```suggestion
     <h:outputLabel value="#{authorImportExport.choose_export_type}" for="exportType" />
```

Reply:
Right, i missed that. Thanks.